content/plugins/descriptions/travis-ci.yaml

```diff
@@ -19,24 +19,12 @@ coverImage:
   alt: 'A screenshot of the Travis CI plugin. It is showing a list of builds for a sample service.'
 
 gettingStarted:
-  - intro: 'Install the plugin in your Backstage instance'
```

Comment:
Users who create their own app rather than using the sample app may have to do this; AFAIK this plugin isn't part of the create app template? It's worth listing all the steps which a user may need to follow and then suggesting they can skip some steps in certain circumstances (i.e. they're running the example app)